### PR TITLE
Cli analytics

### PR DESCRIPTION
**Pull request by shafeeqd959**
Ticket - CS-34690
Version - This changes are going along with rewrites of export and import, so not bumping the versions here

### CODE REVIEW

**Comment by netrajpatel** [comment]
@shafeeqd959 Can you please add the ticket number and version update information in the description of this PR?

File: packages/contentstack-utilities/src/contentstack-management-sdk.ts [hunk 1]
```diff
@@ -43,32 +51,32 @@ export default async (config) => {
...
+
     if (typeof config.branchName === 'string') {
-      option['headers'] = {
-        branch: config.branchName,
-      };
+      option.headers.branch = config.branchName;
+    }
+
+    if (this.analyticsInfo) {
+      option.headers['X-CS-CLI-ID'] = config.analyticsInfo;
```
Comment (netrajpatel) [inline packages/contentstack-utilities/src/contentstack-management-sdk.ts:63]: @shafeeqd959 Are we using `X-CS-CLI` OR `X-CS-CLI-ID`?

**Review by netrajpatel — CHANGES_REQUESTED** [review]

File: packages/contentstack-utilities/src/contentstack-management-sdk.ts (hunk 1 above)
Comment (shafeeqd959) [reply packages/contentstack-utilities/src/contentstack-management-sdk.ts:63]: > @shafeeqd959 Can you please add the ticket number and version update information in the description of this PR?

Sure

Comment (shafeeqd959) [reply packages/contentstack-utilities/src/contentstack-management-sdk.ts:63]: > @shafeeqd959 Are we using `X-CS-CLI` OR `X-CS-CLI-ID`?
I will update this to  `X-CS-CLI`

**Pull request closed without merging**